docs/cloud/rahti/storage/objectstorage.md

```diff
@@ -0,0 +1,280 @@
+# Allas storage in Rahti
+
+More information about [Allas](../../../data/Allas/index.md)  
+
+## Backup to Allas
+
+There are different ways to backup to Allas from Rahti. We will show you two examples:  
+  - The first one is using another pod to copy the content of your persistent volume to Allas.  
+  - The second one is a bash script that you have to execute from your local machine.
+
+## First example: using another pod
+
+For this first example, we assume that we have a `nginx` deployment running with a `PersistentVolumeClaim`. We provide the files for testing purposes.
```

Comment:
For me it was confusing that the word "assume" is used.

What about saying something like: 

"example, we will deploy a `nginx` ...  "